doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb

```diff
@@ -393,98 +439,98 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Integration parameters\n",
-    "sampling_interval = 100\n",
-    "sampling_iterations = 200\n",
-    "\n",
-    "from espressomd.observables import ParticlePositions, RDF\n",
-    "from espressomd.accumulators import Correlator, MeanVarianceCalculator\n",
-    "# Pass the ids of the particles to be tracked to the observable.\n",
-    "part_pos = ParticlePositions(ids=range(N_PART))\n",
-    "# Initialize MSD correlator\n",
-    "msd_corr = Correlator(obs1=part_pos,\n",
-    "                      tau_lin=10, delta_N=10,\n",
-    "                      tau_max=1000 * TIME_STEP,\n",
-    "                      corr_operation=\"square_distance_componentwise\")\n",
-    "# Calculate results automatically during the integration\n",
-    "system.auto_update_accumulators.add(msd_corr)\n",
-    "\n",
-    "# Set parameters for the radial distribution function\n",
-    "r_bins = 100\n",
-    "r_min = 0.0\n",
-    "r_max = system.box_l[0] / 2.0\n",
-    "# Initialize RDF accumulator\n",
-    "rdf_obs = RDF(ids1=system.part[:].id, min_r=r_min, max_r=r_max, n_r_bins=r_bins)\n",
-    "rdf_acc = MeanVarianceCalculator(obs=rdf_obs, delta_N=sampling_interval)\n",
-    "# Accumulate results automatically during the integration\n",
-    "system.auto_update_accumulators.add(rdf_acc)\n",
-    "\n",
-    "# Take measurements\n",
-    "time = np.zeros(sampling_iterations)\n",
-    "instantaneous_temperature = np.zeros(sampling_iterations)\n",
-    "etotal = np.zeros(sampling_iterations)\n",
-    "\n",
-    "for i in range(1, sampling_iterations + 1):\n",
-    "    system.integrator.run(sampling_interval)\n",
-    "    # Measure energies\n",
-    "    energies = system.analysis.energy()\n",
-    "    kinetic_temperature = energies['kinetic'] / (1.5 * N_PART)\n",
-    "    etotal[i - 1] = energies['total']\n",
-    "    time[i - 1] = system.time\n",
-    "    instantaneous_temperature[i - 1] = kinetic_temperature\n",
-    "\n",
-    "# Finalize the correlator and obtain the results\n",
-    "msd_corr.finalize()\n",
-    "msd = msd_corr.result()\n",
-    "\n",
-    "# Finalize the accumulator and obtain the results\n",
-    "rdf = rdf_acc.get_mean()\n",
-    "r = rdf_obs.bin_centers()"
+    "F_TOL = 1e-2\n",
+    "DAMPING = 30\n",
+    "MAX_STEPS = 10000\n",
+    "MAX_DISPLACEMENT = 0.01 * LJ_SIG\n",
+    "EM_STEP = 10"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "solution2": "hidden",
+    "solution2_first": true
+   },
    "source": [
-    "We now use the plotting library <tt>matplotlib</tt> available in Python to visualize the measurements."
+    "**Exercise:**\n",
+    "\n",
+    "* Use <tt>[espressomd.integrate](http://espressomd.org/html/doc/espressomd.html#module-espressomd.minimize_energy).set_steepest_descent</tt> to relax the initial configuration.\n",
```

Comment:
```suggestion
    "* Use [<tt>espressomd.integrate.set_steepest_descent</tt>](http://espressomd.org/html/doc/running.html#steepest-descent) to relax the initial configuration.\n",
```
broken link (`minimize_energy` was removed), not caught in CI because the link was invalid Markdown syntax